docs/README.md

```diff
@@ -1,3 +0,0 @@
-# Welcome to DAOstack Access Control Docs
```

Comment:
Why was this removed?

Reply:
I don't know. I just re-generated the docs...

Reply:
`yarn docs:<..>` only touches `docs/generated_docs`

Reply:
I might have accidentally deleted them... I added them back.